docs/sources/gradle.md

```diff
@@ -21,4 +21,13 @@ To run `licensed` for specific projects in a [multi-build project](https://docs.
 ```yml
 apps:
   - source_path: ./path/to/subproject
+```
+
+### Gradlew 
+
+The `gradle.gradlew` property is used to determine where the `gradlew` executable is. The default location the [configuration root](../configuration/configuration_root.md).
+
+```yml
+gradle:
+  - gradlew: path/from/root/to/gradle/gradlew
 ```
```

Comment:
❤️ thanks for the docs update!!

Reply:
oh 🤦 the nested `gradlew` property isn't an array and shouldn't have the `- ` prefix.  I'll fix the docs in the release PR